homeassistant/components/startca/sensor.py

```diff
@@ -28,19 +32,68 @@
 MIN_TIME_BETWEEN_UPDATES = timedelta(hours=1)
 REQUEST_TIMEOUT = 5  # seconds
 
+
 SENSOR_TYPES = {
-    "usage": ["Usage Ratio", PERCENTAGE, "mdi:percent"],
-    "usage_gb": ["Usage", DATA_GIGABYTES, "mdi:download"],
-    "limit": ["Data limit", DATA_GIGABYTES, "mdi:download"],
-    "used_download": ["Used Download", DATA_GIGABYTES, "mdi:download"],
-    "used_upload": ["Used Upload", DATA_GIGABYTES, "mdi:upload"],
-    "used_total": ["Used Total", DATA_GIGABYTES, "mdi:download"],
-    "grace_download": ["Grace Download", DATA_GIGABYTES, "mdi:download"],
-    "grace_upload": ["Grace Upload", DATA_GIGABYTES, "mdi:upload"],
-    "grace_total": ["Grace Total", DATA_GIGABYTES, "mdi:download"],
-    "total_download": ["Total Download", DATA_GIGABYTES, "mdi:download"],
-    "total_upload": ["Total Upload", DATA_GIGABYTES, "mdi:download"],
-    "used_remaining": ["Remaining", DATA_GIGABYTES, "mdi:download"],
+    "usage": SensorEntityDescription(
+        key="Usage Ratio",
```

Comment:
```suggestion
    SensorEntityDescription(
        key="usage",
        name="Usage Ratio",
```

This applies to all other sensors in this dictionary as well.